src/main/java/org/spongepowered/api/service/pagination/PaginationList.java

```diff
@@ -92,76 +95,141 @@ static Builder builder() {
-     * screen.
+     * Gets the maximum amount of lines that will be sent per page.
+     *
+     * <p>This defaults to the maximum amount of lines that can be displayed
+     * on a source's screen at one time if not specified.</p>
      *
-     * @return The maximum number of lines per page
+     * @return The maximum amount of lines that will be sent per page
      */
     int getLinesPerPage();
 
     /**
-     * Send the constructed pagination list to the given source.
+     * Sends the first page of the constructed pagination list
+     * to the specified message receiver.
      *
-     * @param source The source to send to
+     * @param receiver The receiver to send the first page to
+     * @see PaginationList#sendTo(MessageReceiver, int) to send a specific page
      */
-    void sendTo(MessageReceiver source);
+    default void sendTo(MessageReceiver receiver) {
+        sendTo(receiver, 1);
+    }
 
     /**
-     * Send the constructed pagination list to the specified
-     * {@link MessageChannel}.
+     * Send the specified page of the constructed pagination list
+     * to the specified message receiver.
+     *
+     * <p>A page that is out of bounds will result in a friendly
+     * error message being sent to the receiver.</p>
+     *
+     * <p>Pages start at an index of 1.</p>
      *
-     * @param channel channel to send to
+     * @param receiver The receiver to send the page to
+     * @param page The page to send, starting at an index of 1
      */
-    default void sendTo(MessageChannel channel) {
-        for (MessageReceiver receiver : channel.getMembers()) {
-            this.sendTo(receiver);
+    void sendTo(MessageReceiver receiver, int page);
+
+    /**
+     * Sends the first page of the constructed pagination list to
+     * all {@link MessageReceiver}s within an {@link Iterable}.
+     *
+     * @param receivers The message receivers to send the first page to
+     * @see PaginationList#sendTo(Iterable, int) to send a specific page
+     */
+    default void sendTo(Iterable<MessageReceiver> receivers) {
+        sendTo(receivers, 1);
+    }
+
+    /**
+     * Sends the specified page of the constructed pagination list
+     * all {@link MessageReceiver}s within an {@link Iterable}.
+     *
+     * @param receivers The message receivers to send the page to
+     * @param page The page to send
+     */
+    default void sendTo(Iterable<MessageReceiver> receivers, int page) {
+        checkNotNull(receivers, "The iterable of receivers cannot be null!");
+        for (MessageReceiver receiver : receivers) {
+            sendTo(receiver, page);
         }
     }
 
     /**
+     * Sends the first page of the constructed pagination list to
+     * the members of the specified {@link MessageChannel}.
+     *
+     * @param channel The message channel to send the first page to
+     * @see PaginationList#sendTo(MessageChannel, int) to send a specific page
+     */
+    default void sendTo(MessageChannel channel) {
```

Comment:
That will cause this to be a breaking change as that method already existed. We could do that but since the Javadocs specify it's to the members I think it's okay.